<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDonor</class>
 <widget class="QDialog" name="DialogDonor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>479</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>66666</width>
    <height>888888</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add New AD</string>
  </property>
  <widget class="QGroupBox" name="donorGroupBox">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>361</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Donor Information</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>341</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="donorHorizontalLayout">
     <item>
      <layout class="QGridLayout" name="donorGridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="donorNameLabel">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="donorNameLineEdit">
         <property name="text">
          <string>Donor Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="donorAgeLabel">
         <property name="text">
          <string>Age</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="donorBTLabel">
         <property name="text">
          <string>Blood Type</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="donorAgeSpinBox">
         <property name="maximum">
          <number>130</number>
         </property>
         <property name="value">
          <number>40</number>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="donorAgeSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <spacer name="donorBTSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="donorBTComboBox">
         <item>
          <property name="text">
           <string>O</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>AB</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="donorCharacteristicsGridLayout">
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="donorWeightSpinBox">
         <property name="maximum">
          <double>400.000000000000000</double>
         </property>
         <property name="value">
          <double>60.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="donorGenderComboBox">
         <item>
          <property name="text">
           <string>Male</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Female</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="donorHeightLabel">
         <property name="text">
          <string>Height (m)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="donorWeightLabel">
         <property name="text">
          <string>Weight (kg)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="donorGenderLabel">
         <property name="text">
          <string>Gender</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="donorHeightSpinBox">
         <property name="maximum">
          <double>3.000000000000000</double>
         </property>
         <property name="value">
          <double>1.500000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>391</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="donorInfoTabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="donorHLAInfoTab">
       <attribute name="title">
        <string>HLA Information</string>
       </attribute>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>364</width>
          <height>251</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="donorHLAInfoVerticalLayout">
         <item>
          <layout class="QHBoxLayout" name="donorHLAInfoHorizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="donorMajorHLAVerticalLayout">
             <item>
              <layout class="QGridLayout" name="donorHLAGridLayout">
               <item row="2" column="0">
                <widget class="QLabel" name="donorCWLabel">
                 <property name="text">
                  <string>Donor HLA CW</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="donorALabel">
                 <property name="text">
                  <string>Donor HLA A</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="donorBLabel">
                 <property name="text">
                  <string>Donor HLA B</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="donorDRLabel">
                 <property name="text">
                  <string>Donor HLA DR</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="donorDQLabel">
                 <property name="text">
                  <string>Donor HLA DQ</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="donorHLAA1ComboBox"/>
               </item>
               <item row="0" column="2">
                <widget class="QComboBox" name="donorHLAA2ComboBox"/>
               </item>
               <item row="2" column="2">
                <widget class="QComboBox" name="donorHLACW2ComboBox"/>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="donorHLAB1ComboBox"/>
               </item>
               <item row="2" column="1">
                <widget class="QComboBox" name="donorHLACW1ComboBox"/>
               </item>
               <item row="1" column="2">
                <widget class="QComboBox" name="donorHLAB2ComboBox"/>
               </item>
               <item row="3" column="1">
                <widget class="QComboBox" name="donorHLADQ1ComboBox"/>
               </item>
               <item row="4" column="1">
                <widget class="QComboBox" name="donorHLADR1ComboBox"/>
               </item>
               <item row="3" column="2">
                <widget class="QComboBox" name="donorHLADQ2ComboBox"/>
               </item>
               <item row="4" column="2">
                <widget class="QComboBox" name="donorHLADR2ComboBox"/>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="donorHLAVerticalLayout">
             <item>
              <widget class="QGroupBox" name="donorHLADRLayout">
               <property name="title">
                <string>Donor HLA DR</string>
               </property>
               <widget class="QWidget" name="layoutWidget_5">
                <property name="geometry">
                 <rect>
                  <x>20</x>
                  <y>20</y>
                  <width>59</width>
                  <height>76</height>
                 </rect>
                </property>
                <layout class="QVBoxLayout" name="donorHLADRVerticalLayout">
                 <item>
                  <widget class="QCheckBox" name="donorDR51CheckBox">
                   <property name="text">
                    <string>DR51</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="donorDR52CheckBox">
                   <property name="text">
                    <string>DR52</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="donorDR53CheckBox">
                   <property name="text">
                    <string>DR53</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="donorHLABWLayout">
               <property name="title">
                <string>Donor HLA BW</string>
               </property>
               <widget class="QWidget" name="layoutWidget_6">
                <property name="geometry">
                 <rect>
                  <x>20</x>
                  <y>20</y>
                  <width>55</width>
                  <height>49</height>
                 </rect>
                </property>
                <layout class="QVBoxLayout" name="donorHLABWVerticalLayout">
                 <item>
                  <widget class="QCheckBox" name="donorBW4CheckBox">
                   <property name="text">
                    <string>BW4</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="donorBW6CheckBox">
                   <property name="text">
                    <string>BW6</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="donorHLAAdditionalHorizontalLayout">
           <item>
            <widget class="QLabel" name="donorHLAAdditionalLabel">
             <property name="text">
              <string>Additional Donor Antigens</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="donorHLALineEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <zorder>layoutWidget</zorder>
       <zorder>layoutWidget_7</zorder>
       <zorder>layoutWidget_9</zorder>
       <zorder>verticalSpacer</zorder>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <spacer name="buttonSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogDonor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogDonor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
